miden-lib/src/accounts/faucets/mod.rs

```diff
@@ -28,8 +78,7 @@ const MAX_DECIMALS: u8 = 12;
 /// `distribute` requires authentication. The authentication procedure is defined by the specified
 /// authentication scheme. `burn` does not require authentication and can be called by anyone.
 ///
-/// Public key information for the scheme is stored in the account storage at slot 0. The token
-/// metadata is stored in the account storage at slot 1.
+/// The token metadata is stored in the account storage at slot 0.
 pub fn create_basic_fungible_faucet(
```

Comment:
Ah, this description is actually incomplete and its language is confusing. I think I wanted to express that in the storage layout of the faucet component the metadata is at index 0. But since we're building a full account here and not a component, we should talk about the absolute storage layout of that account. So slot 0 would be the reserved slot, the public key would be at index 1 and the metadata at index 2.

Because we can now talk about absolute and relative indices depending on whether we're talking about an actual `AccountStorage` ("absolute") or about an `AccountComponent` ("relative"), this has some potential for confusion. We should perhaps find some standard language for this, e.g. "absolute" and "relative" or "storage index" and "component index" or something along those lines.